components/edit/configure.tsx

```diff
@@ -313,11 +273,6 @@ const Configure: React.FC<ConfigureProps> = ({ collapsed }) => {
               aria-label="advanced"
               title={<h1>Advanced</h1>}
               startContent={<HiCog />}
-              classNames={{
-                content: collapsed
-                  ? 'py-6 pt-10 h-[500px]'
-                  : 'p-10 pt-6 h-[500px]',
-              }}
```

Comment:
This hack is no longer needed since we have something with space below the models. Previously this made it so that the models had room to display.